.woodpecker/release.yml

```diff
@@ -7,26 +7,21 @@ when:
 
 steps:
   pack-chart:
-    image: quay.io/helmpack/chart-releaser:v1.7.0
+    image: alpine/helm:3.16.4
```

Comment:
This is not an official image from the alpine maintainers. Dont know how trustworthy the author is.